src/Microsoft.AspNetCore.Mvc.RazorPages/IPageActivatorProvider.cs

```diff
@@ -15,13 +16,13 @@ public interface IPageActivatorProvider
         /// </summary>
         /// <param name="descriptor">The <see cref="CompiledPageActionDescriptor"/>.</param>
         /// <returns>The delegate used to activate the page.</returns>
-        Func<PageContext, object> CreateActivator(CompiledPageActionDescriptor descriptor);
+        Func<PageContext, ViewContext, object> CreateActivator(CompiledPageActionDescriptor descriptor);
```

Comment:
Basically, the `Page` gets the `PageContext` and `ViewContext` - the `PageModel` just gets the `PageContext`